src/cpp/tuntap-linux.cpp

```diff
@@ -23,7 +23,9 @@ int init(int fd, bool isTap, char* name) {
         ifr.ifr_flags = IFF_TUN;
     }
 
-    ifr.ifr_flags |= IFF_NO_PI;
+    if(disablePacketInfo)
```

Comment:
Would you please add some test cases for this switch?

Reply:
give me some time, I'll take care of it